credentials/alts/internal/handshaker/handshaker.go

```diff
@@ -25,8 +25,8 @@ import (
 	"fmt"
 	"io"
 	"net"
-	"sync"
 
+	"golang.org/x/sync/semaphore"
```

Comment:
Travis is complaining about `go.mod`. Maybe this is a new dependency? Run `go mod tidy`?

Reply:
We don't need this anymore since there's no dependency on `semaphore`.